### ISSUE TITLE
about identity loss

### ISSUE THREAD

**Issue by karim-moon** [ai-tor/PyTorch-CycleGAN#39]
Hi,
thank you for making this repository and it makes me more easier to learn CycleGAN
But seeing this train code, there is a wandering thing about identity loss

At the paper, there is no identity loss.
So what kind of role does identity loss have in this code?

Thanks

**Comment by karim-moon** [comment]
I asked something stupid,
To make sure the coordinate real image and converted image at the latent space, it used real image identity mapping.
I found original repository about identity mapping code 
Thanks
